app/ts/PreferencesTab/Preferences/TradeOff/TradeOffReferenceCriterionStatement/TradeOffSliderPopover/TradeOffSliderPopover.tsx

```diff
@@ -0,0 +1,82 @@
+import {Grid, Popover, Slider} from '@material-ui/core';
+import {
+  canBePercentage,
+  getPercentifiedValue
+} from 'app/ts/DisplayUtil/DisplayUtil';
+import {determineStepSize} from 'app/ts/PreferencesTab/Elicitation/MatchingElicitation/MatchingElicitationUtil';
+import {SettingsContext} from 'app/ts/Settings/SettingsContext';
+import React, {useContext} from 'react';
+import {TradeOffContext} from '../../TradeOffContext/TradeOffContext';
+
+export default function TradeOffSliderPopover({
```

Comment:
Isn't TradeOffSlider a better name? The popover part seems a bit redundant to me.